src/components/date-picker/DatePicker.vue

```diff
@@ -96,6 +97,7 @@ export default class DatePicker extends Mixins(DateMixin) {
   @Prop({ default: () => [] }) readonly inputRules!: Array<(v) => boolean | string>
   @Prop({ default: false }) readonly disablePicker!: boolean
   @Prop({ default: '' }) readonly initialValue!: string
+  @Prop({ default: true }) readonly showCurrent!: boolean | string
```

Comment:
Does the default value maintain the current behaviour?

Reply:
Yes, the default value on `v-date-picker` for it is `true`: https://v2.vuetifyjs.com/en/api/v-date-picker/#props

Reply:
Perfect answer, thanks! 😃